File: plus_minus_split/plus.py
def main():
    t = int(input())
    for _ in range(t):
        n = int(input())
        a = input()
        ans = 0
        for elem in a:
            if elem == '-':
                ans -= 1
            else:
                ans += 1        
        print(abs(ans))
    
    
if __name__ == "__main__":
    main()

File: plus_minus_split/plus.cpp
#include <iostream>
#include <string>
#include <cstdlib>

using namespace std;

int main()
{
    int t;
    cin >> t;
    for (int i = 0; i < t; i++)
    {
        int n;
        cin >> n;
        string a;
        cin >> a;
        int ans = 0;
        for (int j = 0; j < n; j++)
        {
            if (a.at(j) == '-')
            {
                ans -= 1;
            }

            else
            {
                ans += 1;
            }
        }

        cout << abs(ans) << "\n";
    }
}
